samples/dashboards/basic/your-first-dashboard/demo.css

```diff
@@ -1,4 +1,5 @@
 @import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/dashboards/css/datagrid.css");
```

Comment:
datagrid is not used in this demo, no need to include styling for it.

Reply:
DataGrid is used, Highcharts not.